docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Integral numeric types - C# Reference"
+description: "Learn the range, storage size, and uses for each of the integral numeric types."
+ms.date: 06/25/2019
+f1_keywords:
+  - "byte"
+  - "byte_CSharpKeyword"
+  - "sbyte_CSharpKeyword"
+  - "sbyte"
+  - "short"
+  - "short_CSharpKeyword"
+  - "ushort"
+  - "ushort_CSharpKeyword"
+  - "int_CSharpKeyword"
+  - "int"
+  - "uint"
+  - "uint_CSharpKeyword"
+  - "long_CSharpKeyword"
+  - "long"
+  - "ulong_CSharpKeyword"
+  - "ulong"
+helpviewer_keywords: 
+  - "integral types, C#"
+  - "Visual C#, integral types"
+  - "types [C#], integral types"
+  - "ranges of integral types [C#]"
+  - "byte keyword [C#]"
+  - "sbyte keyword [C#]"
+  - "short keyword [C#]"
+  - "ushort keyword [C#]"
+  - "int keyword [C#]"
+  - "uint keyword [C#]"
+  - "long keyword [C#]"
+  - "ulong keyword [C#]"
+---
+# Integral types  (C# Reference)
+
+The **integral types** are a subset of the **numeric types**. The **numeric types** are a subset of the **simple types**. The integral types are all value types, and generally follow the mathematical rules for integer types. All integral types can be initialized by *literal constants*, for example "24".
```

Comment:
What is the purpose of referencing to simple types (which are not defined (though simple types are defined through the list of types), nor referenced)? Also, that knowledge is not used further in the article, at least explicitly. So, why does a reader need to know that an integral type is a simple type. I'm afraid it might be confusing.

Further, the "all value types" might be understood as "there are no other value types". Also, "mathematical rules" is not clear -> let's reference to operators?
```suggestion
The **integral types** are a subset of the **numeric types**. The **numeric types** are a subset of the **simple types**. All integral types are value types and can be initialized with *literals*, for example "24". All integral types support [arithmetic](../operators/arithmetic-operators.md), [bitwise logical](../operators/bitwise-and-shift-operators.md), [comparison](../operators/comparison-operators.md) and [equality](../operators/equality-operators.md) operators.
```

Reply:
Great edit!

I'm referencing *simple types* because a PR coming soon (after adding floating point, and builtin in reference types) will define simple types using the language spec definition: they can be initialized with literals.